### PR TITLE
docs: add nachrichten to Bubble Tea in the Wild

### PR DESCRIPTION
**Pull request by zMoooooritz**
A little TUI I have written with bubbletea - [nachrichten](https://github.com/zMoooooritz/nachrichten)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -358,6 +358,7 @@ For some Bubble Tea programs in production, see:
 * [mergestat](https://github.com/mergestat/mergestat): run SQL queries on git repositories
 * [meteor](https://github.com/stefanlogue/meteor): a highly customizable conventional commit message tool
 * [mods](https://github.com/charmbracelet/mods): AI on the CLI; built for pipelines
+* [nachrichten](https://github.com/zMoooooritz/nachrichten): stay informed without leaving your command line
```
Comment (meowgorithm) [inline README.md:361]: Mind updating this to something more specific like the following?

> access up-to-date news in German provided by the [Tagesschau](https://www.tagesschau.de/)

**Review by meowgorithm — CHANGES_REQUESTED** [review]
This looks so cool! Happy to merge if you don't mind making the description more specific.

**Comment by maaslalani** [comment]
Thank you for the contribution @zMoooooritz, nice work! ❤️

**Comment by zMoooooritz** [comment]
@meowgorithm Thank you for creating this awesome framework
and thanks to @maaslalani for integrating the improved description
I have been busy the last day and have unfortunately not seen your messages

**Pull request merged**